CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+#Contributing to Friendly Plans.
+##About
+Friendly Plans is an application supporting autism treatment.
+It provides digital version of activity schedules. More about treatment (ENG):
+  
+- http://www.pcdi.org/resources/videos.html
+
+The app is being used and able to download from google play store:
+  
+- https://play.google.com/store/apps/details?id=com.przyjaznydamianek&hl=pl
+
+The article and film shows the usage of app in Institute for Child Development in Gdańsk:
+  
+- Film: https://www.youtube.com/watch?v=zI0ma_XnmCc 
+    
+- Article: http://iwrd.pl/pl/fundacja/aplikacja-przyjazny-plan-dostepna
+
+The project was started by students from Gdańsk University of Technology, however it needs improvements.
+##License
+The project is developing by people who want to make life easier for others and Friendly plans are going to be free to use and download forever for everybody. The license is GPL 3.0.
```

Comment:
... who want to make other people's life easier so Friendly Plans are and are going to be free to use, download and develop for everybody forever. ...